projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -200,8 +200,8 @@ export class ClrDatagridWidgetObject extends WidgetObject<ClrDatagrid> {
      * Gives the header above the grid.
      */
     get gridHeader(): string | undefined {
-        const headerElement = this.root.nativeElement.previousSibling;
-        return headerElement.nodeType !== 8 ? headerElement.textContent : undefined;
+        const headerElement = this.findElement('h3', this.root.parent);
```

Comment:
As you mentioned, this feels weird because it is wrong. This widget object is only supposed to wrap the ClrDatagrid. Our Grid contains extra stuff, we really need a separate DataGrid object since it contains a `ClrDataGrid`

Let me know if you'd like to chat about this. I think it just needs to have a public method to get its clr data grid and one more method to get the grid header.

This looks like it should be defined using `getText`